src/Microsoft.DotNet.Interactive.Tests/Utility/TemporaryDirectory.cs

```diff
@@ -4,31 +4,58 @@
 using System;
 using System.IO;
 
-namespace Microsoft.DotNet.Interactive.Tests.Utility
+namespace Microsoft.DotNet.Interactive.Tests.Utility;
+
+public class TemporaryDirectory : IDisposable
```

Comment:
Is this still needed? If not is there a reason not to use the `DisposableDirectory` class we already have?

Reply:
removed in favour of the one already present